app/src/test/java/org/mozilla/fenix/ext/ListTest.kt

```diff
@@ -58,9 +82,9 @@ class ListTest {
         file2.createNewFile()
         file3.createNewFile()
 
-        val item1 = DownloadItem("71", "filepath.txt", filePath1, "71 Mb", "text/plain", DownloadState.Status.COMPLETED)
-        val item2 = DownloadItem("72", "filepath2.txt", filePath2, "71 Mb", "text/plain", DownloadState.Status.COMPLETED)
-        val item3 = DownloadItem("73", "filepath3.txt", filePath3, "71 Mb", "text/plain", DownloadState.Status.COMPLETED)
+        val item1 = DownloadItem("71", "url", "filepath.txt", filePath1, "71 Mb", "text/plain", DownloadState.Status.COMPLETED)
+        val item2 = DownloadItem("72", "url", "filepath2.txt", filePath2, "71 Mb", "text/plain", DownloadState.Status.COMPLETED)
+        val item3 = DownloadItem("73", "url", "filepath3.txt", filePath3, "71 Mb", "text/plain", DownloadState.Status.COMPLETED)
```

Comment:
You could've copied what you had above and get those named parameters to future proof yourself

Reply:
Addressed